src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteManagedOrganizationUserAccountCommand.cs

```diff
@@ -98,6 +114,7 @@ public async Task DeleteUserAsync(Guid organizationId, Guid organizationUserId,
             }
         }
 
+        await DeleteManyAsync(users);
```

Comment:
Also, `DeleteManyAsync` should only receive users that have passed validation. (i.e. where they have an empty string in their `results` entry.)